change/@microsoft-teams-js-7ebef344-0a7e-4f6d-8a75-7adec5ad1f57.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "initial",
```

Comment:
>initial [](http://example.com/codeflow?start=14&length=7)

Can you elaborate on the change here, since this text will show-up in the public CHANGELOG?